src/common/utils/stringHelpers.js

```diff
@@ -24,6 +16,7 @@ import {
   TYPE_VIDEO,
   UUID_REGEX,
 } from 'common/constants'
+import { TYPE_DOCUMENT } from '../constants/searchParams'
```

Comment:
I might be missing something or looking in the wrong file, but I don't see this added to the searchParams file as a part of this PR, and neither on the current code on dev. Could you please check ?

Reply:
Great catch. Thanks :)